tox.ini

```diff
@@ -30,7 +31,9 @@ deps =
     pytest==4.6.3
     pytest-cov==2.7.1
     pytest-helpers-namespace==2019.1.8
+    pytest-html>=1.21.1
     pytest-mock==1.10.4
+    pytest-molecule>=1.0.1
```

Comment:
We don't use it.

Reply:
> Installs pytest-molecule plugin in order to verify that the presence of the plugin does not negatively impact pytest execution.

You can do that manually. We don't need to add it here?

Reply:
I will remove it.